report_qweb_encrypt/static/src/js/report/action_manager_report.js

```diff
@@ -0,0 +1,96 @@
+// © 2017 Creu Blanca
+// License AGPL-3.0 or later (https://www.gnuorg/licenses/agpl.html).
+odoo.define("report_qweb_encrypt.Dialog", function (require) {
+    "use strict";
+
+    var ActionManager = require("web.ActionManager");
+    var Dialog = require("web.Dialog");
+    var core = require("web.core");
+
+    var _t = core._t;
+
+    var EncryptDialog = Dialog.extend({
```

Comment:
I still can't get the password dialog to popup. I have zero knowledge about JS. Can someone take a look?

Reply:
Fixed by this, something changed from v12.

https://github.com/OCA/reporting-engine/compare/104472445acef310d720c25acd1ff934061b40dd..6e79bfd5cf80bd41a617931fb1f96651789ba1f9